medusa/schedulers/download_handler.py

```diff
@@ -45,6 +45,7 @@ class ClientStatusEnum(Enum):
 
     SNATCHED = 0  # 0
     PAUSED = 1  # 1
+    STOPPED = 1
```

Comment:
the only thing i'm not sure here, is if we need another download handler state. What benefit brings this?

Reply:
Semantics to go along with qbittorent deciding to rebrand pause to stop, and the other parts adding "stop" wouldn't work was my interpretation.

Reply:
Also a quick effort to not affect other torrent programs that don't conform to whatever qbittorrent maintainers silly decision, and retain backwards compatibility with pre 5.0.0 qbittorrent versions.

at least that was my intention

Reply:
@p0psicles i remember now
https://github.com/pymedusa/Medusa/pull/11857/files#diff-28c840b0e4fc446765e43a85e5cb6aed6a874eee15753e980007e12c86fa6096R243
This line would not work unless a new state of stopped was defined, and i didn't want to cause any unwanted effects to existing ecosystem, only support qbittorrent 5.0+

Reply:
(i am first now registering i made a typo in that function too)

Reply:
i summarized this better here: https://github.com/pymedusa/Medusa/issues/11844#issuecomment-2395316482

Reply:
I had time to deploy this on a physical machine again, and removed the changes to download_handler.py